test/api/test-ot-face.c

```diff
@@ -150,7 +150,7 @@ test_font (hb_font_t *font, hb_codepoint_t cp)
   hb_draw_funcs_destroy (funcs);
 #endif
 
-  hb_font_get_glyph_contour_point (font, 0, 0, NULL, NULL);
```

Comment:
Not my assumption but https://github.com/harfbuzz/harfbuzz/blob/2dda6dd744b192616c65ca4aa5fce8e90fd0ff30/src/hb-font.cc#L416